app/helpers/safe-markup.js

```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
-export function safeMarkup(params/*, hash*/) {
+/**
+  * safeMarkup helper. Returns a string that will not be HTML escaped by Handlebars.
+  *
+  * @method safeMarkup
+  * @param {string} string
+  * @return {Object} Returns Handlebars.SafeString
+  */export function safeMarkup(params/*, hash*/) {
```

Comment:
hmm, trying to get `yarn run docs` to work - seeing `error Command "docs" not found.`